src/components/avatar_name/index.tsx

```diff
@@ -10,14 +10,21 @@ const AvatarName: React.FC<AvatarName> = ({
   className, address, name, imageUrl, href = ACCOUNT_DETAILS,
 }) => {
   const classes = useStyles();
+  let avatarName = name;
+
+  if (name === '""') {
+    avatarName = address;
+  } else if (name.length === 1) {
+    avatarName = address;
+  }
 
   return (
     <Link href={href(address)}>
       <a>
         <div className={classnames(className, classes.root)}>
           <Avatar address={address} imageUrl={imageUrl} />
           <Typography variant="body1">
-            {name || address}
+            {name ? avatarName : address}
           </Typography>
         </div>
       </a>
```

Comment:
So if the name = "q" or "a" we will show address instead?
```
else if (name.length === 1)
```

Aside from the above comment, this works but I think we can do better. 
The title was extremely misleading and it's 100% my fault. 

Since most of the issue is coming from the validators itself so we can probably fix this without touching the `AvatarName` component at all.

Method 1:
In `src/recoil/validators/hooks.ts` we can handle it here. On line 49 `const moniker = name || '';`
We can add some more checks such as:

```
let moniker = '';
if (name.replace(/ /g, '').length && name !== '""') { // check that it has length and it does not equal `'""'`
  moniker = name;
}
```

I would prefer to do it on this level in the event that we might actually want to `'"'''` on another account. Just not validators. Setting the assignment here gives us more control on a local level as oppose to the global component level

Method 2:
Basically what you did here.

Comments:
You can decide how you want to proceed as long as you can back it up. Just want to show there may be another method to this

Reply:
Updated to method 1.